### PR TITLE
Upgrade to spago 0.93.3 and generate lockfile

### PR DESCRIPTION
**Pull request by thomashoneyman**

### CODE REVIEW

File: spago.lock [hunk 1]
```diff
@@ -0,0 +1,1695 @@
+workspace:
+  packages:
+    registry-app:
+      path: node_modules/registry-app
```
Comment (thomashoneyman) [inline spago.lock]: Nooooooooooooooo!

Comment (f-f) [reply spago.lock]: Well, that's disappointing 😂

Comment (f-f) [reply spago.lock]: How did we solve it before?

Comment (thomashoneyman) [reply spago.lock]: I don't know why node_modules keeps getting picked up as the paths. I'll delete the directory and rebuild the lockfile.

Comment (thomashoneyman) [reply spago.lock]: It's as-of-yet still unsolved.

Comment (f-f) [reply spago.lock]: Released a 0.93.4, let's see how that one goes..

**Pull request closed without merging**